src/components/LoadingIndicator/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { ProgressIndicator } from "@nypl/design-system-react-components";
+
+const LoadingIndicator = (
```

Comment:
Could you expose the props for `ProgressIndicator` for this component? Feel free to reference the [PageHeading.tsx](https://github.com/NYPL/nypl-library-card-app/pull/434/files/2e1fd5b3009500205c645dc888bb2de6438174ca#diff-7dfafc89b3f47e1691a22a94bb10912c0f52d411dcf9cfe8eb8c35b6ef1d9e65) component